gbus/builder/builder.go

```diff
@@ -107,6 +112,7 @@ func (builder *defaultBuilder) Build(svcName string) gbus.Bus {
 	if builder.usingPingTimeout {
 		gb.DbPingTimeout = builder.dbPingTimeout
 	}
+	gb.Deduplicator = implementation.NewDeduplicator(svcName, builder.deduplicationPolicy, gb.TxProvider, builder.deduplicationRetentionAge, gb.Log())
```

Comment:
i think that from a logging perspective it would be better we pass in the log of the worker and not of the bus as the worker log will have additional context information that gets passed to the deduper